src/SarifCli.FunctionalTests/TestData/DoNotUseFriendlyNameAsRuleId/RuleIdMatchesFriendlyName_Expected.sarif

```diff
@@ -0,0 +1,83 @@
+{
```

Comment:
The validator produces this output file for the log file that _does_ violate the rule. There is one `result`.